packages/frontend/src/features/catalog/components/CatalogMetadata.tsx

```diff
@@ -63,14 +63,14 @@ export const CatalogMetadata: FC<React.PropsWithChildren<Props>> = ({
 
             <Table>
                 <thead>
-                    <th>
-                        <td>field name</td>
-                        <td>type</td>
-                    </th>
+                    <tr>
+                        <th>field name</th>
+                        <th>type</th>
+                    </tr>
                 </thead>
                 <tbody>
                     {data.fields?.map((field) => (
-                        <tr key="">
+                        <tr key={field.name}>
```

Comment:
table names are unique, field names are not... but I guess they are unique per table, so this should be ok... Thanks for fixing, I guess it was me who left this empty

Reply:
Yea, I figured this would be unique enough here. Could always add table name too but 'shrug'